CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
-# Release 4.12.2 (June 7, 2024)
+# Unreleased
 
 - Add `typing_extensions.get_annotations`, a backport of
   `inspect.get_annotations` that adds features specified
-  by PEP 649. Patch by Jelle Zijlstra.
+  by PEP 649. Patches by Jelle Zijlstra and Alex Waygood.
```

Comment:
This was in the changelog section for 4.12.2, but it didn't actually make it into that release ;)